.storybook/custom-loader.js

```diff
@@ -14,9 +15,17 @@ module.exports = () => {
         return !excludePlugins.includes(requestedPlugin);
       });
 
+      const existingPresets = /** @type {Required<import('@babel/core').ConfigItem[]>} */ (cfg.options.presets || []);
+      const presetsToInclude = existingPresets.filter(plugin => {
+        const requestedPreset = /** @type {NonNullable<typeof plugin['file']>} */ (plugin.file).request;
+
+        return !excludePresets.includes(requestedPreset);
+      });
+
       return {
         ...cfg.options,
         plugins: pluginsToInclude,
+        presets: presetsToInclude,
```

Comment:
@Hotell you asked in offline if it works:

![image](https://user-images.githubusercontent.com/14183168/150801386-a6029596-ae29-429b-8ceb-6bb890889d32.png)

- Yes, it works and keeps styles without compilation as it was before with `@fluentui/babel-make-styles`.